sunpy/timeseries/timeseriesbase.py

```diff
@@ -138,25 +126,43 @@ def __init__(self, data, meta=None, units=None, **kwargs):
 
 # #### Attribute definitions #### #
 
+    @property
+    def data(self):
+        """
+        The original pandas DataFrame.
+        """
+        return self.data
```

Comment:
I may be getting something wrong, but `data` and `meta` aren't already public attributes as they were? 
In any case, why if I do the following doesn't work?
```python
class MyClass(object):
    def __init__(self, data, meta):
        self.data = data
        self.meta = meta 
    @property
    def _data(self):
        return self.data
```
Should the `self.data` be renamed as `self._data` (making it private)? Wouldn't then you also need some setter? Is that to do some data sanity?

Reply:
Yeah, I realised that what I did doesn't actually work, I'll revert it...